Unix/installbuilder/conf/omiserver.conf

```diff
@@ -33,7 +35,7 @@ httpsport=0
 #NtlmCredsFile=/etc/opt/omi/.creds/ntlm
 
 ##
-## <NICKNAME> -- set the value of nickname.
+## <NICKNAME> -- set the value of nickname for omi related directory and file path.
```

Comment:
What, exactly, is a "NICKNAME"? What does it do? Here I am, working with OMI for a long time, and even I don't know this. Clarify in this comment please.

Reply:
here is a customized configuration for omi directories and files, like omiserver or omiagent path, omi log directory and path, ssl directory and ssl keypem path, etc.